src/app/AddCard.tsx

```diff
@@ -22,23 +29,14 @@ const AddCard = ({ onStep }: StepProps) => {
             <button onClick={() => onStep && onStep({ step: 'list' })}>&lt;</button>
             &nbsp; 카드 추가
           </h2>
-          <CardBox />
+          <CardBox isOpenCardType={setIsShowModal} />
```

Comment:
```suggestion
const openModal = () => setIsShowModal(true)
const closeModal = () => setIsShowModal(false)

<CardBox openModal={openModal} closeModal={closeModal}/>
```

일반적으로 `is`, `has` 등의 prefix는 boolean값을 의미하며 함수의 경우 동사로 시작하는 편이 코드 읽기에 수월해요